Ocarina/OcarinaManager.swift

```diff
@@ -238,8 +238,13 @@ extension OcarinaManager: URLSessionDataDelegate {
         }
         
         var html: HTMLDocument? = nil
+
         if let data = data {
```

Comment:
`If let data {...}` should be enough or does this need to support an older swift version?